### DISCUSSION TITLE
When will the text_embedding feature of baidu Wenxin yiyan be available for integration?

### DISCUSSION THREAD

**Discussion by shiliu2023** [langgenius/dify#6224] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

When will the text_embedding feature of baidu Wenxin yiyan be available for integration?

### 2. Additional context or comments

When will the text_embedding feature of baidu Wenxin yiyan be available for integration?

### 3. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by takatost** [comment]
Hey @shiliu2023, if ur interested, u can check out https://github.com/langgenius/dify/blob/main/api/core/model_runtime/README.md to integrate the embedding model capabilities. Thx a lot for ur support! 🙌
